composer/trainer/trainer.py

```diff
@@ -2552,7 +2552,6 @@ def eval(
 
         """
         if eval_dataloader is not None:
```

Comment:
not sure if this is the linter acting up but probably best to `git checkout dev -- trainer/trainer.py` if no changes intended.